### PR TITLE
chore!: update from module template

### PR DESCRIPTION
**Pull request by waxb**

### CODE REVIEW

File: README.tfdoc.hcl [hunk 1]
```diff
@@ -99,7 +99,7 @@ section {
 
         variable "role" {
           type        = string
-          description = "The role that should be applied. Note that custom roles must be of the format `[projects|organizations]/{parent-name}/roles/{role-name}`."
+          description = "The role that should be applied. Note that custom roles must be of the format `[projects|organizations]/{parent-name}/roles/{role-name}`. Must be set if `policy_binding` is unset"
```
Comment (mariux) [inline README.tfdoc.hcl:102]: please make it multi line string for better readability..

File: README.tfdoc.hcl [hunk 2]
```diff
@@ -115,7 +115,7 @@ section {
 
         variable "policy_bindings" {
           type           = list(policy_binding)
-          description    = "A list of IAM policy bindings."
+          description    = "A list of IAM policy bindings. Must be set if `role` is unset"
```
Comment (mariux) [inline README.tfdoc.hcl:118]:
```suggestion
          description    = "A list of IAM policy bindings. If set `role` is ignored and a policy_binding is created."
```

File: README.tfdoc.hcl (hunk 1 above)
Comment (mariux) [reply README.tfdoc.hcl:102]: it's `policy_bindings`

**Review by mariux — CHANGES_REQUESTED** [review]

File: README.tfdoc.hcl [hunk 3]
```diff
@@ -115,7 +119,7 @@ section {
 
         variable "policy_bindings" {
           type           = list(policy_binding)
-          description    = "A list of IAM policy bindings."
+          description    = "A list of IAM policy bindings. If set `role` is ignored and a policy_binding is created."
```
Comment (zied-elouaer) [inline README.tfdoc.hcl:122]:
```suggestion
          description    = "A list of IAM policy bindings. If set, `role` is ignored and a policy_binding is created."
```

**Review by zied-elouaer — CHANGES_REQUESTED** [review]

**Review by zied-elouaer — APPROVED** [review]
LGTM

**Review by mariux — APPROVED** [review]
lgtm

**Pull request merged**